### PR TITLE
docs/nia: Update status API docs to table format

### PR DESCRIPTION
**Pull request by mkam**

### CODE REVIEW

File: website/content/docs/nia/api/status.mdx [hunk 1]
```diff
@@ -21,18 +21,24 @@ This endpoint currently returns the overall status information for all tasks.
...
-    * `critical` - (int) The number of tasks that have a 'critical' health status
-    * `unknown` - (int) The number of tasks that have an 'unknown' health status
-  * `enabled` - Summary count of how many tasks are enabled or disabled
-    * `true` - (int) The number of tasks that are enabled
-    * `false` - (int) The number of tasks that are disabled
+#### Response Fields
 
+| Name	| Type	 | Description        |
+| ----- | ------ | ------------------ |
+|`status` | object |  Summary count of how many tasks have each health status value.
```
Comment (karl-cardenas-coding) [inline website/content/docs/nia/api/status.mdx:34]:
```suggestion
|`status` | object |  The summary count of how many tasks that each have a health status value.
```
Cleaner?

File: website/content/docs/nia/api/status.mdx [hunk 2]
```diff
@@ -75,37 +81,50 @@ Task health status value is determined by the success or failure of all stored [
...
-* `task_name` - (string) Name that task is configured with in Consul-Terraform-Sync.
-* `status` - (string) Values are "successful", "errored", "critical", or "unknown". This is determined by the success or failure of all stored events on the network infrastructure update process for the task, as described earlier.
-* `enabled` - (bool) Whether the task is enabled or not.
-* `services` - (list[string]) List of the services configured for the task.
-* `providers` - (list[string]) List of the providers configured for the task.
-* `events_url` - (string) Relative URL to retrieve the event data stored for the task.
-* `events` - [(list[Event])](/docs/nia/api#event) - List of stored events that inform the task's status. See section below for information on event data. This field is only included in the response upon request by setting the `?include=events` parameter. The relative URL for the request to include events can be retrieved from the `events_url` field.
+| Name	      | Type	  | Description       |
+| ----------- | ------ | ------------------ |
+|`task_name`  | string | Name that task is configured with in Consul-Terraform-Sync.
```
Comment (karl-cardenas-coding) [inline website/content/docs/nia/api/status.mdx:103]:
```suggestion
|`task_name`  | string | Name of the task as configured in Consul-Terraform-Sync.
```
Cleaner?

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good, just two minor suggestions/questions.

**Review by lornasong — APPROVED** [review]
looks good to me! table format reads ✨

File: website/content/docs/nia/api/status.mdx (hunk 1 above)
Comment (mkam) [reply website/content/docs/nia/api/status.mdx:34]: I think this implies that not all tasks have a health status, which isn't the case. Maybe something like `The summary of how many tasks have a given health status value`?

**Pull request merged**